free.fr known issue (#3834)
okta/okta-developer-docs: packages/@okta/vuepress-site/docs/guides/custom-email/main/index.md

1. Click the email icon to the right of the email template that you have customized. A list shows the sender and receiver of the email.
2. Click **Send test email**.

## Known Issues

> - End users at free.fr email addresses may not receive emails delivered by Okta. This is due to free.fr enforcing non-industry standard email processing. Okta's Org API allows you to [retry blocked](/docs/reference/api/org/#email-address-bounces-operations)) emails but the free.fr inbox provider continues to block retries. There is no known workaround beyond asking the end user to retrigger the email tih a different email address.

Reviewer: ```suggestion
- Users with free.fr email addresses may not receive emails delivered by Okta. This is due to free.fr enforcing non-industry standard email processing. The Okta Org API allows you to [retry blocked](/docs/reference/api/org/#email-address-bounces-operations) emails, but the free.fr inbox provider continues to block retries. There is no known workaround beyond asking the user to retrigger the email with a different email address.
```
